concepts/interfaces/about.md

```diff
@@ -43,15 +43,15 @@ func (s Stats) SomeOtherMethod() {
 }
 ```
 
-For implementing the interface, it does not matter whether the method has a value or poiter receiver.
+For implementing the interface, it does not matter whether the method has a value or pointer receiver.
 (Revisit the [methods concepts][concept-methods] if you are unsure about those.)
```

Comment:
JFYI: the link seems to be not workin in the UI. Is this intentional?

Reply:
Thanks for mentioning this. I will take a look independent of this PR and make another one if needed.